api/swagger.yml

```diff
@@ -2207,38 +2332,37 @@ paths:
       responses:
         200:
           description: object metadata
-          schema:
-            $ref: "#/definitions/object_stats"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ObjectStats"
         401:
-          $ref: "#/responses/Unauthorized"
+          $ref: "#/components/responses/Unauthorized"
         404:
-          description: path or branch not found
-          schema:
-            $ref: "#/definitions/error"
+          $ref: "#/components/responses/NotFound"
```

Comment:
Here we have a less informative description of the response.  Consider having multiple versions of `NotFound`, each with different documentation "path or branch [or repo] not found", "branch [or repo] not found", "repo not found".

Reply:
We only have one 404 - the error description is a resource not found as a general way to describe any of the resources in the way of the request.
A single request can return errors of repository, branch, and reference not found. So the current solution is a more general way to describe it.